pymc3/distributions/continuous.py

```diff
@@ -1654,6 +1655,93 @@ def logcdf(self, value):
         )
 
 
+class AsymmetricLaplace(Continuous):
+    r"""
+    Asymmetric-Laplace log-likelihood.
+
+    The pdf of this distribution is
+
+    ..math::
+        {f(x|\\b,\kappa) =
+            \left({\frac{\\b}{\kappa + 1/\kappa}}\right)\,e^{-(x)\\b\,s\kappa ^{s}}}
+        where s = sgn(x)
+
+    Parameters
+    ----------
+    b:
+        Scale parameter (b > 0)
+    kappa:
+        Symmetry parameter (kappa > 0)
+
+    See also: https://en.wikipedia.org/wiki/Asymmetric_Laplace_distribution
+    """
+
+    def __init__(self, b, kappa, testval=0.0, *args, **kwargs):
+        self.b = tt.as_tensor_variable(b)
+        self.kappa = tt.as_tensor_variable(kappa)
+
+        super().__init__(*args, **kwargs, testval=testval)
+
+    def _random(self, b, kappa, size=None):
+        if size is not None:
```

Comment:
I think this condition should return false when size is passed as an empty tuple. Let's go more explicit here by using `to_tuple` function.
```suggestion
        size = to_tuple(size)
        if len(size) != 0:
```